src/series/errorBase.js

```diff
@@ -0,0 +1,99 @@
+import d3 from 'd3';
+import fractionalBarWidth from '../util/fractionalBarWidth';
+
+export default function() {
+
+    var xScale = d3.time.scale(),
+        yScale = d3.scale.linear(),
+        yHigh = d3.functor(0),
+        xHigh = function(d, i) { return d.date; },
+        yLow = d3.functor(0),
+        xLow = function(d, i) { return d.date; },
+        xValue = function(d, i) { return d.date; },
+        yValue = function(d, i) { return d.y; },
+        barWidth = fractionalBarWidth(0.75),
+        xValueScaled = function(d, i) {
+            return xScale(xValue(d, i));
+        };
+
```

Comment:
I am following the pattern seen in ohlc here, I will refactor to single dimension and then it may change anyway. Thanks for suggestion.